frontend/src/reducks/store/store.js

```diff
@@ -1,22 +1,26 @@
-import { applyMiddleware, combineReducers, configureStore } from 'redux';
```

Comment:
What is the purpose of changing this file?

Reply:
Sir if we use CreateStore it gives us a warning to use configure store

Reply:
![image](https://user-images.githubusercontent.com/89973801/181291348-9afa4a3b-6cc4-486d-9c57-736cae4f3515.png)